.claude/skills/bob-hunt/SKILL.md

```
@@ -249,27 +251,45 @@ After completion, call `bounty_transition_phase({ target_domain, to_phase: "VERI
-Read `bounty_read_verification_round(round='final').data`. If no result has `reportable: true`, do not stop: call `bounty_read_evidence_packs({ target_domain: "[domain]" })` to confirm `skipped: true`, then continue through GRADE and REPORT so the session gets a durable SKIP grade and no-findings report. If final reportables exist, spawn the evidence agent before GRADE:
+Agent(subagent_type: "balanced-verifier", name: "balanced", prompt: "Session: ~/bounty-agent-sessions/[domain]. Egress profile: [egress_profile]. First call bounty_read_verification_context({ target_domain }). If v1, read brutalist and preserve the legacy cascade. If v2, do not read brutalist or adjudication; use current_attempt_id and snapshot_hash, pass verification_replay context on replay tools, cover exactly snapshot findings, then write only through bounty_write_verification_round(round='balanced').")
+```
+3. After both complete, call `bounty_read_verification_context({ target_domain })` again. Require brutalist and balanced statuses to be `current: true`; retry a missing/invalid worker once.
+4. Call `bounty_build_verification_adjudication({ target_domain })`. Use only its returned `.data.adjudication_plan_hash` and adjudication payload; do not compute diffs in prose or ask the final verifier to compute diffs.
+5. Launch the final verifier with current attempt ID, snapshot hash, and `adjudication_plan_hash`. The final verifier must consume the adjudication plan hash and write `round="final"` with `adjudication_plan_hash`.
+```
+Agent(subagent_type: "final-verifier", name: "final-verify", prompt: "Session: ~/bounty-agent-sessions/[domain]. Egress profile: [egress_profile]. First call bounty_read_verification_context({ target_domain }). If v2, consume the current adjudication_plan_hash from the orchestrator/context, do not compute diffs, pass verification_replay context on replay tools, and write round='final' with verification_attempt_id, verification_snapshot_hash, and adjudication_plan_hash. If v1, read balanced and use the legacy final cascade.")
+```
+
+After final verification in either branch, read `bounty_read_verification_round({ target_domain: "[domain]", round: "final" }).data`. For v2, require `.data.current === true` and no `stale` flag. If no result has `reportable: true`, do not stop: call `bounty_read_evidence_packs({ target_domain: "[domain]" })` to confirm `skipped: true`, then call `bounty_transition_phase({ target_domain, to_phase: "GRADE" })` and continue through GRADE and REPORT so the session gets a durable SKIP grade and no-findings report. If final reportables exist, spawn the evidence agent before GRADE:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)